perfmetrics/scripts/testing_on_gke/examples/utils/gsheet_test.py

```diff
@@ -0,0 +1,76 @@
+"""This file defines unit tests for functionalities in utils.py"""
+
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
nit: Test contains two Copyright 2018, but not the main file. Can we correct it?